### PR TITLE
Calculate water kind with CSV

### PR DESCRIPTION
**Pull request by zerebubuth**
This is the first of a series of PRs dealing with #646 by taking each layer and making the `min_zoom` and `kind` calculation come from a CSV file instead. This PR does it for the water layer.

I ran this through a series of migrations and the tests all pass... which probably says that we need more tests rather than I didn't break anything!

Connects to #646.

@rmarianski could you review, please?

### CODE REVIEW

**Comment by rmarianski** [comment]
:+1:

**Comment by zerebubuth** [comment]
@rmarianski apologies - I made a bunch more changes. Could you look those over too, please?

File: data/migrations/v0.10.0-line.sql [hunk 1]
```diff
@@ -1,3 +1,17 @@
 UPDATE planet_osm_line
 SET mz_water_min_zoom = mz_calculate_min_zoom_water(planet_osm_line.*)
 WHERE mz_calculate_min_zoom_water(planet_osm_line.*) IS NOT NULL;
+
+CREATE INDEX CONCURRENTLY new_planet_osm_line_water_geom_9_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 9;
+CREATE INDEX CONCURRENTLY new_planet_osm_line_water_geom_12_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 12;
+CREATE INDEX CONCURRENTLY new_planet_osm_line_water_geom_15_index ON planet_osm_line USING gist(way) WHERE mz_water_min_zoom <= 15;
```
Comment (rmarianski) [inline data/migrations/v0.10.0-line.sql]: Do we need to create the indexes concurrently?

File: spreadsheets/kind/pois.csv [hunk 2]
```diff
@@ -101,6 +103,5 @@ station,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${13},station
 *,*,*,*,*,*,*,*,*,*,*,adit,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${16},adit
 *,*,*,*,*,*,*,*,*,*,*,water_well,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${17},water_well
 *,*,*,*,*,*,*,*,*,*,*,*,saddle,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${14},saddle
-*,*,*,*,*,*,*,*,*,*,*,*,dune;geyser;sinkhole,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${15},"${""natural""}"
-*,*,*,*,*,*,*,*,*,*,*,*,hot_spring,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${16},hot_spring
+*,*,*,*,*,*,*,*,*,*,*,*,dune;sinkhole,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,${15},"${""natural""}"
```
Comment (rmarianski) [inline spreadsheets/kind/pois.csv]: Did you mean to put these changes and the test update on a different branch?

Comment (rmarianski) [inline spreadsheets/kind/pois.csv]: Never mind. For some reason I assumed that this was all water specific.

**Comment by rmarianski** [comment]
:+1:

**Pull request merged**